CMakeLists.txt

```diff
@@ -211,6 +211,17 @@ if(NOT ${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
         set(CMAKE_C_FLAGS   "${CMAKE_C_FLAGS} -funroll-loops -Wall -Wno-unknown-pragmas  ")
         set(CMAKE_SHARED_LINKER_FLAGS "-Wl,--no-undefined")
         message(STATUS "Additional default options were set for LLVM/Clang compiler")
+      elseif(${CMAKE_CXX_COMPILER_ID} MATCHES Intel)
+        if (CMAKE_CXX_COMPILER_ID STREQUAL "IntelLLVM")
+          # "IntelLLVM": Intel's NextGen LLVM compiler (icx)
+          set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
+        else (CMAKE_CXX_COMPILER_ID STREQUAL "IntelLLVM")
+          # "Intel": Intel's Classic compiler (icc)
+          message(FATAL_ERROR "Intel Classic compiler is not supported on Windows")
+        endif()
+        set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS}")
```

Comment:
```suggestion
```

Reply:
I am trying to fix compilation, and I think is flag realted, I will remove it if finally it reamins the same